buildSrc/src/main/groovy/airbyte-connector-acceptance-test.gradle

```diff
@@ -1,10 +1,27 @@
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 
+
 class AirbyteConnectorAcceptanceTestPlugin implements Plugin<Project> {
+    
+    // Change this value to 'dev' if you want to use a locally built connector acceptance test image
+    String CONNECTOR_ACCEPTANCE_TEST_IMAGE = 'latest'
```

Comment:
Could this be exposed as an argument to the command somehow? I think this means when working on acceptance test changes, we won't be able to test our changes against connectors easily

Reply:
An ENVIRONMENT variable might be a good way to handle this

Reply:
I've added an option to the `/test` command: `connector-acceptance-test-version`, which defaults to latest.
It's then passed to gradle as a project property via env var with `ORG_GRADLE_PROJECT_connectorAcceptanceTestVersion`.
So, on a Connector Acceptance Test PR, we can run `/test connector=connectors/source-pokeapi connector-acceptance-test-version=dev` to use a locally built CAT.

Reply:
Here's an example workflow to which I've passed connector-acceptance-test-version=dev : https://github.com/airbytehq/airbyte/actions/runs/4222475489